docs/src/components/parts/FirstSection.js

```diff
@@ -33,12 +33,18 @@ export default function FirstSection() {
             </p>
           </div>
 
-          <div className="mt-12 !z-10 mb-6">
+          <div className="mt-12 !z-10 mb-6 space-x-4">
             <a
-              href="../docs/setup/download"
+              href="../docs/quickstart"
               className="text-2xl !z-10 !no-underline hover:text-white py-2 px-6 font-bold btn bg-indigo-800 hover:bg-indigo-900 border-indigo-800 hover:border-indigo-900 text-white rounded-md"
             >
-              Download Now
+              Quickstart
+            </a>
+            <a
+              href="../docs/setup/download"
+              className="text-2xl space-x-4 !z-10 !no-underline hover:text-white py-2 px-6 font-bold btn bg-indigo-800 hover:bg-indigo-900 border-indigo-800 hover:border-indigo-900 text-white rounded-md"
```

Comment:
```suggestion
              className="text-2xl !z-10 !no-underline hover:text-white py-2 px-6 font-bold btn bg-indigo-800 hover:bg-indigo-900 border-indigo-800 hover:border-indigo-900 text-white rounded-md"
```

You shouldn't need the extra class since it is set on the parent already

Reply:
Good catch. Leftover from experimentation.